Plugins/Gerrit/FormGerritPublish.Designer.cs

```diff
@@ -205,7 +247,11 @@ private void InitializeComponent()
         private System.Windows.Forms.TextBox _NO_TRANSLATE_Topic;
         private System.Windows.Forms.TextBox _NO_TRANSLATE_Reviewers;
         private System.Windows.Forms.Label labelReviewers;
-        private System.Windows.Forms.ComboBox PublishType;
+        private System.Windows.Forms.ComboBox _NO_TRANSLATE_PublishType;
```

Comment:
Could you please clarify why you have renamed these controls?

Reply:
Actually, I'm not sure if it needs to be renamed. As I understand, it is _NO_TRANSLATE_ because it is also the ComboBox Name. I renamed it to keep consistency with all another controls, like: Topic, Branch, Remotes, Reviewers. They all shouldn't have _NO_TRANSLATE_ prefix, am I right?

Reply:
We name certain controls with `_NO_TRANSLATE` prefix to exclude those from translations, e.g. those that hold value that should not be translated, e.g. part of git commands, urls, etc.

Looking at the original LOC value of `PublishType`, it should be translated, hence no prefix should be required.
Unless there is a new requirement, which isn't totally obvious.

Reply:
In this case Topic, Branch, Remotes and Reviewers also can be translated.

Reply:
Values in these textboxes are user-provided, so they are not translated